appcues/src/main/java/com/appcues/ui/presentation/AppcuesViewModel.kt

```diff
@@ -26,14 +29,20 @@ import kotlinx.coroutines.flow.collectLatest
 import kotlinx.coroutines.launch
 
 internal class AppcuesViewModel(
-    private val renderContext: RenderContext,
+    private val binding: PresentationBinding,
```

Comment:
seems like nice cleanup, maybe name `presentationBinding` to match type, as `binding` is pretty generic term